client/interview-controller.js

```diff
@@ -0,0 +1,72 @@
+// Description:
+//   A Hubot script which helps hrs to control interviews.
+//
+// Commands:
+//  begin group Huntflow Reloaded
+//    begin admin
+//      hubot удалить интервью - remove an interview from the list
```

Comment:
Let's replace the description with `removes the non-expired interview of the specified candidate`.